WikiDrips/UIImageExtensions.swift

```diff
@@ -10,15 +10,15 @@ import UIKit
 
 extension UIImage {
     
-    func image(withInitials: String?, in bounds: CGRect) -> UIImage {
+    static func image(withInitials initials: String?, in bounds: CGRect) -> UIImage {
         let scale = UIScreen.main.scale
         
         UIGraphicsBeginImageContextWithOptions(bounds.size, false, scale)
         defer {
             UIGraphicsEndImageContext()
         }
         
-        guard let initials = withInitials, let context = UIGraphicsGetCurrentContext() else {
+        guard let monogram = initials, let context = UIGraphicsGetCurrentContext() else {
```

Comment:
FYI, Eldar and I don't do this, the `guard` statement create a new scope so that 
`guard let initials =  initials` creates a new local variable `initials` that is unwrapped and valid after the guard. You're not over-writing the original optional `initials`. Also, you'll probably never use the optional version after you unwrap it in a `guard` otherwise you wouldn't use a `guard`. There are a lot of optional unwrapping in Swift so you'll be challenged to come up with new names for all of them.